api/src/main/resources/workbench.yaml

```diff
@@ -525,6 +525,35 @@ paths:
          schema:
           $ref: '#/definitions/ErrorReport'
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/retrieve-localize-files:
```

Comment:
I might just call this (and related methods) "localize-files"... the fact that you have to look up what files to retrieve before they are localized is a detail that doesn't have to be reflected in the name?

Also, you can probably remove localize-notebooks and the code / UI that calls it.

Reply:
Removing the localize-notebook call would result in reverting the whole logic for displaying notebook files and "PUSH TO SERVER" functionality.

Would the name localize-all-files make more sense?

Reply:
Sure, "localize-bucket-files" is fine.

My assumption is that we're going to automatically sync files between GCS and the cluster and vice-versa; and as such there is no need for UX where you manually push files to the bucket. Researchers shouldn't have to think about pushing files.

(Anyway, if we're not sure about that yet, you can leave the localize-notebooks stuff in for now.)

Reply:
Fixed. I have just change the Api name